packages/builder/src/actions/rounds.ts

```diff
@@ -134,7 +127,7 @@ export const loadRound =
         pointer: v2Round.applicationMetadataCid,
       },
       applicationMetadata,
-      programName,
+      programName: v2Round.roundMetadata.name,
```

Comment:
Ah okie this is cleaner !
@0xKurt I think the reason why we had this confusion was we used a random metadata in our script which didn't have name in the round metadata (which caused the breakage)